cg/cli/workflow/balsamic/base.py

```diff
@@ -228,67 +187,54 @@ def start(
     analysis_type: str,
     panel_bed: str,
     priority: str,
-    dry: bool,
+    dry_run: bool,
 ):
     """Start full workflow for CASE ID"""
     LOG.info(f"Starting analysis for {case_id}")
-    context.invoke(link, case_id=case_id)
-    context.invoke(config_case, case_id=case_id, panel_bed=panel_bed, dry=dry)
-    context.invoke(
-        run,
-        case_id=case_id,
-        analysis_type=analysis_type,
-        priority=priority,
-        run_analysis="--run-analysis",
-        dry=dry,
-    )
-
-
-@balsamic.command("store")
-@ARGUMENT_CASE_ID
-@OPTION_DRY
-@OPTION_ANALYSIS_TYPE
-@click.pass_context
-def store(context: click.Context, case_id: str, analysis_type: str, dry: bool):
-    """Generate Housekeeper report for CASE ID and store in Housekeeper"""
-    LOG.info(f"Storing analysis for {case_id}")
-    context.invoke(report_deliver, case_id=case_id, analysis_type=analysis_type, dry=dry)
-    context.invoke(store_housekeeper, case_id=case_id)
+    try:
+        context.invoke(resolve_compression, case_id=case_id, dry_run=dry_run)
+        context.invoke(link, case_id=case_id)
+        context.invoke(config_case, case_id=case_id, panel_bed=panel_bed, dry_run=dry_run)
+        context.invoke(
+            run,
+            case_id=case_id,
+            analysis_type=analysis_type,
+            priority=priority,
+            run_analysis="--run-analysis",
+            dry_run=dry_run,
+        )
+    except DecompressionNeededError as e:
+        LOG.error(e.message)
 
 
 @balsamic.command("start-available")
 @OPTION_DRY
 @click.pass_context
-def start_available(context: click.Context, dry: bool):
-    """Start full workflow for all available BALSAMIC cases"""
-    balsamic_analysis_api = context.obj["BalsamicAnalysisAPI"]
-    exit_code = EXIT_SUCCESS
-    for case_id in balsamic_analysis_api.get_cases_to_analyze():
+def start_available(context: click.Context, dry_run: bool = False):
```

Comment:
I would prefer to create a submodule and split these functions into multiple files instead of one long file with many functions

Reply:
Next time!